pkgs/by-name/ma/manticoresearch/package.nix

```
@@ -18,12 +18,12 @@
 let
   columnar = stdenv.mkDerivation (finalAttrs: {
     pname = "columnar";
-    version = "c21-s10"; # see NEED_COLUMNAR_API/NEED_SECONDARY_API in Manticore's cmake/GetColumnar.cmake
+    version = "c25-s15-k3"; # see NEED_COLUMNAR_API/NEED_SECONDARY_API/NEED_KNN_API in Manticore's cmake/GetColumnar.cmake
     src = fetchFromGitHub {
       owner = "manticoresoftware";
       repo = "columnar";
       rev = finalAttrs.version;
-      hash = "sha256-TGFGFfoyHnPSr2U/9dpqFLUN3Dt2jDQrTF/xxDY4pdE=";
+      hash = "sha256-WD84JVmHrXJkAre7fgsnWkGA84WSuw6Pa2EXyE/k4zA=";
```

```suggestion
      hash = "sha256-im7bjf7h+if/H+Gpq7SnSURKWtTJQRnCpATebTBKKy0=";
```
as reported by ofborg